docs/source/models/cause.rst

```diff
@@ -171,29 +174,321 @@ Common data sources for cause models
+   :widths: 20 30 30
+   :header-rows: 1
+
+   * - Measure
+     - Definition
+     - Uses
+   * - `Prevalence`_
+     - Proportion of population with a given condition.
+     - Initialize cause model states.
+   * - `Birth Prevalence`_
+     - Proportion of all live births born with a given condition.
+     - Initialize neonatal cause model states.
+   * - `Incidence`_
+     - Number of new cases of a given condition per person-year.
+     - Estimate transition rates.
+   * - `Remission`_
+     - Number of recovered cases from a given condition per person-year.
+     - Estimate transition rates.
+   * - `Disability Weights`_
+     - Proportion of full health not experienced due to disability associated
+       with a given condition.
+     - Measure disability attributed to cause model states.
+   * - `Cause-specific Mortality`_
+     -
+     -
+   * - `Excess Mortality`_
+     -
+     -
 
-Incidence
-+++++++++
+Prevalence
+++++++++++
+
+Prevalence is defined as the **proportion of a given population that possesses
+a given condition or trait** at a given time-point.
 
-Birth prevalence
+  For example, the prevalence of obesity in the United States was
+  approximately 40% in 2016.
+
+When a *time-frame* (such as 2016, i.e. 1/1/16 to 12/31/16) instead of a
+*time-point* (such as 1/1/16) is reported, it is commonly assumed that the
+reported prevalence represents the prevalence of the *midpoint* of
+that time-frame (7/1/16 is the midpoint for the time frame of all of 2016).
+However, this may not always be the case and it should be considered when
+relevant.
+
+Prevalence data can be used to **initialize cause model states** and
+represents the **probability that a simulant will begin the simulation in a
+given state.**
+
+  For example, the probability that a simulant in a model of obesity in the
+  United States beginning in 2016 will begin the simulation as obese is 0.4 or
+  40%.
+
+Birth Prevalence
 ++++++++++++++++
 
-Remission
+Birth prevalence is defined as the **proportion of live births in a given
+population that possess a given condition or trait at birth.**
+
+  For example, the birth prevalence for cleft lip in the United States in 2006
+  was 10.6 per 10,000 live births, or 0.106%.
+
+Birth prevalence data can be used to **initialize neonatal cause model
+states** and represent the **probability that a simulant who is born during
+the simulation will be born into a given neonatal cause model state.**
+
+  For example, the probability that a simulant born during a simulation of
+  cleft lip in the United States in 2006 is 0.00106, or 0.106%.
+
+Incidence
 +++++++++
 
-Prevalence
+Incidence rates are defined as the **number of new cases of a condition that
+occur per person-year of the at-risk population (individuals without
+condition).** Specifically, the at-risk population can be represented as
+`1 - condition prevalence`.
+
+  For example, the incidence of multiple sclerosis (MS) in the United States
```

Comment:
Only true for immortal pops.  Maybe three examples
1. in a pop who never dies
2. in a realistic pop
3. thinking about age specific when you have a realistic pop and different size age bins (e.g. 7-28 day olds).

Reply:
I think these calculations actually compute the **cumulative incidence** (according to Wikipedia, this is the number of new cases during a period divided by the number of at risk subjects at the beginning). As a function of time, this is the same as the cumulative distribution function for the random variable representing the time someone gets the condition. 

The incidence we want here (the one the model needs) is the **incidence rate**. The relationship is
(cumulative incidence) = 1 - exp[-(incidence rate) * duration],
assuming a constant rate over the time period.

So I think maybe the issue is not necessarily about simpler vs more complicated scenarios, but rather just being clear about which thing we're talking about.

Reply:
They do compute the cumulative incidence, but that's what GBD calls incidence.  We use it as essentially our best guess of the incidence hazard.

Reply:
Does the GBD always use cumulative incidence? For example, the Measles section in the YLD appendix specifically refers to the incidence rate:
> We modelled measles incidence in count space with a mixed-effects linear regression of case notifications on routine measles vaccination rates (first- and second-dose measles-containing vaccines) and coverage of supplementary immunisation activities (SIAs). More precisely, log-transformed **incidence rates** were regressed on the log of the proportion unvaccinated with first- and second-dose measles-containing vaccine, and additional SIA coverage lagged by one, two, three, four, and five years, with super-region, region, and country-level random effects:
Yij =beta_0+beta_1 MCV1ij +beta_2 MCV2ij +beta_a3 SIAaij +uj+eij,